common-util/src/main/java/org/sunbird/common/models/util/KeyCloakRsaKeyFetcher.java

```diff
@@ -0,0 +1,109 @@
+package org.sunbird.services.sso.impl;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.RSAPublicKeySpec;
+import java.util.Base64;
+import java.util.Base64.Decoder;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.PropertiesCache;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+
+/**
+ * @author   github.com/iostream04
+ *  This class will connect to key cloak server and fetch the sso_publicKey using http call.
+ */
+
+public class KeyCloakRsaKeyFetcher  {
+	
+	
+	public PublicKey getPublicKeyFromKeycloak(String url, String realm) {
```

Comment:
Please fix indentation we use an indent of 4-spaces for Java files.

Reply:
okay, i will do the respective changes